common/beerocks/bwl/dwpal/base_wlan_hal_dwpal.h

```diff
@@ -20,6 +20,15 @@ extern "C" {
 #include <chrono>
 #include <memory>
 
+#ifdef USE_LIBSAFEC
```

Comment:
Since you moved it to a header file, how about deleting this from the implementation files?
#665